cypress/e2e/oss/onboarding.test.ts

```diff
@@ -14,7 +15,7 @@ describe('Onboarding Redirect', () => {
 describe('Onboarding', () => {
   beforeEach(() => {
     cy.flush()
-
+    cy.wrapEnvironmentVariablesForOss()
```

Comment:
should we move this into the flush function? or no?

Reply:
The flush function is used before the cloud e2e tests as well and the variables aren't the same at this point for OSS and Cloud because on the cloud side it gets the default org dynamically after logging in.  So I think these are better kept separate as the aliases on the cloud side happen on signin and I wouldn't want something to get added to OSS that isn't overwritten by cloud causing confusion down the line.

I do see a potential for a big performance gain in the flush function though so keep your eyes peeled for that :)